dds/DCPS/transport/rtps_udp/RtpsUdpInst.cpp

```diff
@@ -255,6 +259,17 @@ RtpsUdpInst::host_addresses() const {
   return addresses;
 }
 
+void
+RtpsUdpInst::update_locators(const RepoId& remote_id,
+                             const TransportLocatorSeq& locators)
+{
+  TransportImpl_rch impl = this->impl();
```

Comment:
```suggestion
  TransportImpl_rch impl = impl();
```

Reply:
Generates compiler errors.

Reply:
Well maybe the automatic variable needs a different name?